101-app-service-regional-vnet-integration/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Deploy an app service with regional VNet integration",
+  "description": "This template allows you to deploy an app service plan and a basic Windows web app, with regional VNet integration enabled to a newly created virtual network",
+  "summary": "This template allows you to deploy an app service plan and a basic Windows web app, with regional VNet integration enabled to a newly created virtual network",
+  "githubUsername": "johndowns",
+  "dateUpdated": "2020-03-23",
+  "environments": [
```

Comment:
this sample looks fairly straightforward - why no Azure Gov?

Reply:
The verification deployment failed when deploying to Azure Gov. I'm not sure why, but possibly the feature is so new that it hasn't been rolled out to sovereign clouds yet.

Reply:
What was the error?  (usually it will say RP is not available in the region or words to that effect)...  If you don't have it handy we can just run CI again.

Reply:
It didn't give an error, just timed out. Looks like you are running it again anyway.

Reply:
@bmoore-msft As you can see, [the deployment to Azure Gov is stalled](https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=20610&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=7bf03a3b-69d5-5660-244f-21e634d6903f).

Reply:
Closing this as it looks like you've re-instated my original environment list.